djpsa/utils.py

```diff
@@ -51,3 +51,25 @@ def redis_lock(lock_name, timeout, blocking_timeout):
             lock.release()
         except redis.exceptions.LockNotOwnedError:
             pass
+
+
+class DjPSASettings:
+    # TODP never actually needed to be a class for CW and AT, so maybe
```

Comment:
What do you suggest then? Simpler is always preferred.

Reply:
Oh haha, missed that TODO because it was a typo. I dunno, if its never needed to be a class the `DjPSASettings` may as well be canned and its just a module level function: `get_settings`.